.github/workflows/test.yml

```diff
@@ -26,6 +26,10 @@ jobs:
     name: ${{ matrix.os }} Py-${{ matrix.python }} IPFS-${{ matrix.ipfs }}
     steps:
       - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
I am not sure if this should be here because this workflow is executed on push as well, for which I do not know what would be the PR head.

Reply:
Agreed! This was added during the discovery phase before I realized that the absolute reference to the URI using the master branch was embedded in the test case. I will remove this change to test.yml.

Reply:
Removed the change to test.yml in f50d06f (below)